File: app/src/main/java/com/artmurka/artmurkaapp/data/model/pojo/itemlist/orders/Payment.java

package com.artmurka.artmurkaapp.data.model.pojo.itemlist.orders;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Payment {

    @SerializedName("id")
    @Expose
    private String id;
    @SerializedName("topay")
    @Expose
    private String topay;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getTopay() {
        return topay;
    }

    public void setTopay(String topay) {
        this.topay = topay;
    }

}

File: app/src/main/java/com/artmurka/artmurkaapp/data/model/pojo/itemlist/orders/Payment.kt
package com.artmurka.artmurkaapp.data.model.pojo.itemlist.orders

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Payment {

    @SerializedName("id")
    @Expose
    var id: String? = null
    @SerializedName("topay")
    @Expose
    var topay: String? = null

}
